assets/docs/sayakpaul/mixer_b16_i1k_classification/1.md

```diff
@@ -0,0 +1,50 @@
+# Module sayakpaul/mixer_b16_i1k_classification/1
+
+MLP-Mixer fine-tuned on ImageNet-1k.
```

Comment:
Here and elsewhere: WDYT about adding the type to the sentence so it's immediately clear whether it's a B-16, a B-32 or an L-16 mixer?

Reply:
Sure. 12 changes :3 

Will do it right away.

Reply:
@WGierke done.